themes/ddbasic/sass/components/node/news.scss

```diff
@@ -268,6 +268,19 @@
         top: 0;
         right: 7px;
       }
+      .field-item {
+        display: inline-block;
+        margin-right: 5px;
+        &::after {
+          content: ',';
```

Comment:
This inserts a comma with margin to the right to simulate a space. This could be improved/simplified further by [adding an actual space after the comma](https://stackoverflow.com/a/40963855). This would also require less overriding for the last element.